apps/web/src/server/views/applications/case-project-team/project-team-remove.njk

```diff
@@ -0,0 +1,45 @@
+{% extends "../layouts/applications.layout.njk" %}
+
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/table/macro.njk" import govukTable %}
+
+{% set serviceName = "Project team" %}
+{% set pageTitle = "Remove team member" %}
+
+{% block pageTitle %}
+	{{ serviceName }} – {{ pageTitle | default('GOV.UK') }}
+{% endblock %}
+
+{% block beforeContent %}
+	<aside>
+		{{ govukBackLink({
+			text: "Back to Project team",
+			href: 'project-team'|url({caseId: caseId })
+	}) }}
+	</aside>
+{% endblock %}
+
+{% block pageHeading %}{% endblock %}
+
+{% block pageContent %}
+	<h1 class="govuk-heading-l govuk-!-margin-bottom-5">
+		{{ pageTitle }}
+	</h1>
+
+	<form method="post" action="" novalidate="novalidate" class="govuk-!-margin-top-6">
+
+		{{govukTable({
+			head: [
+								{ text: "Name" },
+								{ text: "Email" }
+							],
+							rows: [[
+								{text: projectTeamMember.givenName + '' + projectTeamMember.surname },
```

Comment:
Should there be a space instead of an empty string between those?
```suggestion
								{text: projectTeamMember.givenName + ' ' + projectTeamMember.surname },
```